### ISSUE TITLE
nimforum doesn't build with nim@#devel - nilseqs error

### ISSUE THREAD

**Issue by bketelsen** [nim-lang/nimforum#204]
Environment: Mac OS X

```
choosenim show (built 9/19/2018)
   Channel: devel
   Version: #devel
      Path: /Users/bketelsen/.choosenim/toolchains/nim-#devel
```

```
Nim Compiler Version 0.18.1 [MacOSX: amd64]
Compiled at 2018-09-19
Copyright (c) 2006-2018 by Andreas Rumpf

active boot switches: -d:release
```

Building with the suggested `jester@0.4.0` fails with header errors.  Jester issues suggest moving to `jester@#head`.  
New error is ` ... forum.nim(79, 46) Error: usage of 'isNil' is a user-defined error`

IRC users suggested that changes in sequence behavior is the root:
```
<PMunch> @bketelsen, in the newest version of Nim sequences and strings can't be nil
<PMunch> You can go back to the old behaviour with --nilseqs:on, or you can use a stable version of Nim
```

Confirmed that compiling with `nim c --nilseqs:on src/forum.nim` builds successfully.

**Comment by dvn0** [comment]
Is there a compiler commit that I can give to choosenim which will enable me to build this? 0.18.0 is too old, and 0.19.0 and greater are too new.

**Comment by dom96** [comment]
Here you go: https://github.com/nim-lang/nimforum/blob/master/.travis.yml#L35

**Comment by dvn0** [comment]
Thanks. I tried with that commit in choosenim, but it failed. I created an issue here: https://github.com/dom96/choosenim/issues/103

**Comment by jyapayne** [comment]
This should be also fixed with #225
